RELEASE.md

```diff
@@ -0,0 +1,60 @@
+# Release Process
```

Comment:
@villander this is the description how to use it

Reply:
Thank you so much 💯